charts/values.yaml

```diff
@@ -17,11 +17,9 @@ cloudprovider: "digitalocean"
 
 kubernetesVersion: ""
 
-# general RKE options
-rke:
-  # enable local auth endpoint
-  localClusterAuthEndpoint:
-    enabled: false
+# enable local auth endpoint
+localClusterAuthEndpoint:
+  enabled: false
   # specify fqdn of local access endpoint
```

Comment:
Should probably realign indentation here.

Reply:
Hi @jakefhyde, I am unable to determine how we want to proceed with the indentations. Per my understanding, we expect the `localClusterAuthEndpoint` to be independent of `.Values.rke` and hence should be `.Values.localClusterAuthEndpoint` - As `cacerts` & `fqdn` are subfields of it, I have let the comments indented  for these.
Can you please help me with what I am missing?

Reply:
Sorry should've clarified, the lines 23-31 were indented assuming 1 level of indentation. Now that the `rke` field is removed, they just need to be unindented to match lines 21-22, so they can be uncommented when in use and have the expected indentation level.

Reply:
Just to confirm, the `rke` field is removed entirely right? If so, you decreased indentation for the `enabled` field to reflect that, I would expect you to do the same for the `fqdn` and `caCerts` fields.

Reply:
Yes, that's correct. 
Thanks! I see it now & updated the indentation accordingly. PTAL

Reply:
OH looks like it was right before I left this initial comment haha, whoops. Nice job, and thank you for your patience!